README.md

```diff
@@ -5,20 +5,41 @@
 [![Docker Build Status](https://img.shields.io/docker/automated/degauss/jfs_aggregated_data_report)](https://hub.docker.com/repository/docker/degauss/jfs_aggregated_data_report/tags)
 [![GitHub release (latest by date)](https://img.shields.io/github/v/release/degauss-org/jfs_aggregated_data_report)](https://github.com/degauss-org/jfs_aggregated_data_report/releases)
 
-## DeGAUSS example call
+## Instructions for Use
 
-If `my_address_file_geocoded.csv` is a file in the current working directory with coordinate columns named `lat` and `lon`, then
+### Open the Starter App
 
-```sh
-docker run --rm -v ${pwd}:/tmp degauss/jfs_aggregated_data_report:5.0.0 my_address_file_geocoded.csv
-```
+1.  In Docker Desktop, search for "Degauss/shiny_starter"
+
+    A. "Pull" the container
+
+2.  Once the container is in your local Docker Desktop, find it and click "Run"
+
+    A. In the popup box, expand the options and in the space marked "Local Host", type in "3838" and click Run
+
+3.  Navigate to the Containers page and in your line for the shiny_starter, click to button to open in browser
+
+4.  Now that you're in the app, you can upload your .csv file
+
+5.  After the upload, the app will show you a sample of your original file and the prepared file for use in the next program
+
+6.  Click the button to download the prepped file
+
+    A. Save the downloaded file to a known location that you can navigate to (such as "/Downloads")
```

Comment:
For mac/linux, the download path would be `~/Downloads`; writing to `/Downloads` would be very different! Should we make this windows specific since it is for Kris anyway?  I'm not sure what the path would be for a Windows machine with the home directory structure.  We could ask Stephen to check this part out and help fill in the details.